doc/source/examples/extended_examples/index.rst

```diff
@@ -23,6 +23,8 @@ with other programs, libraries, and features in development.
 +----------------------------+--------------------------------------------------------------------------------------------+
 | :ref:`gui_example`         |  Demonstrates how to create your own GUI app with Python and PySide6                       |
 +----------------------------+--------------------------------------------------------------------------------------------+
+| :ref:`hpc_ml_ga_example`   |  Demonstrates how to use PyMAPDL in a high-performance computing system managed by SLURM.  |
```

Comment:
This description has a period, and the previous one does not. Add a period to the previous description ore remove the one here.